README.md

```diff
@@ -540,38 +535,52 @@ options:
   -t ENTRY_TYPE, --entry-type ENTRY_TYPE
                         Name of the type (from the CDDL) to interpret the data
                         as.
+  --default-max-qty DEFAULT_MAX_QTY, --dq DEFAULT_MAX_QTY
+                        Default maximum number of repetitions when no maximum
+                        is specified. It is only relevant when handling data
+                        that will be decoded by generated code. If omitted, a
+                        large number will be used.
+  --sanitize            Whether to sanitize/unsanitize CBOR-only values. When
```

Comment:
I guess sanitize is not a inclusive term, (sanitycheck was replaced with twister in an effort to better inclusiveness), also I don't think its a good name anyway, as you are ensuring YAML/JSON compatibility, which an extra set of limitations, not enforced by CBOR itself.

My suggestions would be:
```
--ensure_alternate_serialization
--restricted
--ensure_yaml
--yaml_compatible
```

Reply:
Thanks, changed to `--yaml-compatibility`